ovs/extensions/hypervisor/apis/kvm/sdk.py

```diff
@@ -387,6 +387,55 @@ def clone_vm(self, vmid, name, disks, mountpoint):
         return self.create_vm_from_template(name, source_vm, disks, mountpoint)
 
 
+    def create_volume(self, location, size):
+        """
+        Create volume using truncate
+        @param location: Location on filesystem
+        @param size: size in GB
+        """
+        if self.ssh_client is None:
+            self.ssh_client = SSHClient(self.host, username='root')
```

Comment:
I see this check quite often. Why isn't the client initialized in the sdk's __init__ method? If not preferred to do so, maybe a decorator ```@connected()``` would make things easier to read?